Wow, that's interesting! Sorry about that, should've tested it first.

```bash
vip-go-api on  trunk [📝] took 10s 
[ 11:10:35 ] eslint /Users/uzer/Documents/Development/vip-go-api/src/inactive-users/inactive-users.e2e.spec.ts --fix

/Users/uzer/Documents/Development/vip-go-api/src/inactive-users/inactive-users.e2e.spec.ts
   59:3   warning  Disabled test                                                                      jest/no-disabled-tests
  158:21  warning  Identifier name 'inactiveUserWithIgnoreInactiveCheckInThePast' is too long (> 40)  id-length

✖ 2 problems (0 errors, 2 warnings)
```

before
```ts
import alerts from '@lib/utils/alerts';
import {
	createTestApp,
	createTestingModule,
	TestApp,
} from '@src/shared/test-utils/modules.test-utils';
import { InactiveUsersRepository } from '@src/inactive-users/inactive-users.repository';
import { expectErrorResponseBody } from '@src/shared/test-utils/expect.test-utils';
import { AppModule } from '@src/app.module';
import { createApiUser, createApiUserToken } from '@fixtures/api-user/api-user';
import { createClient } from '@fixtures/client/client';
import moment from 'moment/moment';
import testUtils from '@fixtures/global-setup/test-utils';
import { UsersRepository } from '@src/users/users.repository';
import { EventsService } from '@src/events/events.service';

```


after
```ts
import moment from 'moment/moment';

import { createApiUser, createApiUserToken } from '@fixtures/api-user/api-user';
import { createClient } from '@fixtures/client/client';
import testUtils from '@fixtures/global-setup/test-utils';
import alerts from '@lib/utils/alerts';
import { AppModule } from '@src/app.module';
import { EventsService } from '@src/events/events.service';
import { InactiveUsersRepository } from '@src/inactive-users/inactive-users.repository';
import { expectErrorResponseBody } from '@src/shared/test-utils/expect.test-utils';
import {
	createTestApp,
	createTestingModule,
	TestApp,
} from '@src/shared/test-utils/modules.test-utils';
import { UsersRepository } from '@src/users/users.repository';
```

--- a/__fixtures__/javascript.js
+++ b/__fixtures__/javascript.js
@@ -1,4 +1,5 @@
 import { unusedEs6Import } from './stub/unusedEs6Import';
+import { Buffer } from 'node:buffer';
 
 const cp = require( 'child_process' );
 
@@ -46,3 +47,7 @@ export function shadow() {
 const implicitCoercion = !! 'heyyy';
 
 nonExistent();
+
+export function base64encode( str ) {
+	return Buffer.from( str ).toString( 'base64' );
+}

--- a/__fixtures__/typescript.ts
+++ b/__fixtures__/typescript.ts
@@ -1,4 +1,5 @@
 import { unusedEs6Import } from './stub/unusedEs6Import';
+import { Buffer } from 'node:buffer';
 
 export function add( one: number, two: number, three: String ): number {
 	return one + two;
@@ -34,3 +35,7 @@ function someCoolDecorator( totallyRadArgument, _value ): typeof totallyRadArgum
 
 @someCoolDecorator
 export class EmptyClass {}
+
+export function base64encode( str: string ): string {
+	return Buffer.from( str ).toString( 'base64' );
+}

--- a/__tests__/config-export.js
+++ b/__tests__/config-export.js
@@ -1,4 +1,11 @@
+/**
+ * External dependencies
+ */
 const fs = require( 'fs' );
+
+/**
+ * Internal dependencies
+ */
 const configs = require( '../configs' );
 
 const configNames = Object.keys( configs ).sort();

--- a/__tests__/rules/no-async-foreach.ts
+++ b/__tests__/rules/no-async-foreach.ts
@@ -1,5 +1,12 @@
-import rule from '../../rules/no-async-foreach';
+/**
+ * External dependencies
+ */
 import { Linter, RuleTester } from 'eslint';
+
+/**
+ * Internal dependencies
+ */
+import rule from '../../rules/no-async-foreach';
 
 const ruleTester = new RuleTester();
 const parserOptions: Linter.ParserOptions = { ecmaVersion: 8 };

--- a/configs/javascript.js
+++ b/configs/javascript.js
@@ -134,6 +134,22 @@ module.exports = {
 		'import/no-unresolved': 'error',
 		'unused-imports/no-unused-imports': 'warn',
 
+		// Enforce external / internal import groups and alphabetical ordering.
+		'import/order': [
+			'error',
+			{
+				'newlines-between': 'always',
+				alphabetize: {
+					order: 'asc',
+				},
+				groups: [
+					[ 'builtin', 'external' ],
+					[ 'index', 'internal', 'object', 'parent', 'sibling' ],
+					[ 'type' ],
+				],
+			},
+		],
+
 		// Enforce Unix linebreaks. Included here and not in "formatting" since it
 		// is not controversial and helps with interchange.
 		'linebreak-style': [ 'error', 'unix' ],

--- a/utils/is-package-installed.js
+++ b/utils/is-package-installed.js
@@ -1,5 +1,12 @@
-const debugLog = require( './debug-log' );
+/**
+ * External dependencies
+ */
 const findPackageJson = require( 'find-package-json' );
+
+/**
+ * Internal dependencies
+ */
+const debugLog = require( './debug-log' );
 
 // Get a list of all package.json files in the current directory tree, ascending
 // up the tree from the current directory. Note that this code behaves differently

